src/jit/emitxarch.cpp

```diff
@@ -4040,6 +4041,43 @@ void emitter::emitIns_R_R_R_I(
     emitCurIGsize += sz;
 }
 
+static bool isAvxBlendv(instruction ins)
```

Comment:
Just wanted to confirm. These instructions qualify under `Is4ByteAVXInstruction` and `Is4ByteSSEInstruction`, but they still need to be handled specially, correct?

Reply:
Yes.